### DISCUSSION TITLE
增加登录页reCaptchaV2验证

### DISCUSSION THREAD

**Discussion by jdjwzx233** [Qexo/Qexo#316] [Ideas]
总是被判定为机器人。。。
希望能支持reCaptchaV2。
或者也可以在V3没通过时使用V2

**Comment by sysfox** [comment]
@am-abudu 

我觉得Cloudflare的Turnstile也可以考虑加入Qexo配置，在国内相比reCaptcha更容易设置
https://developers.cloudflare.com/turnstile/get-started/migrating-from-recaptcha/

**Comment by am-abudu** [comment]
已添加reCaptchaV2。Turnstile还在计划中
